### DISCUSSION TITLE
OIDC plugin

### DISCUSSION THREAD

**Discussion by karolkielecki** [saleor/saleor#9469] [Announcements]
Hi Saleors! 

We'd like to announce that the OIDC plugin is now public. Learn more here: https://docs.saleor.io/docs/3.x/developer/available-plugins/openid-connect

---

PS If you like what we do, please [like & share](https://twitter.com/getsaleor/status/1510941440746180608) :)

**Comment by manojLondhe** [comment]
Fantastic news. Like, share - done! @karolkielecki 

In which Saleor-core [release](https://github.com/saleor/saleor/releases) this will be included? Meanwhile can we clone and use `main` branch?

**Reply by karolkielecki** [reply]
@manojLondhe Thanks for your support! Not sure when it will be included--let me ask the team. For now, it's been merged into the `main` branch --> https://github.com/saleor/saleor/pull/9406. Generally speaking, we recommend using the latest stable releases. You can use the main branch, but it's not recommended and at your own risk :)

**Reply by karolkielecki** [reply]
@manojLondhe It should be added to Saleor 3.2. There is no release date--but most likely this month.

**Reply by manojLondhe** [reply]
thanks for letting know.

**Comment by wellzenon** [comment]
Amazing! I was trying to make one plugin for external auth, hahahaha. 

Does the OIDC plugin work with more than one provider simultaneously?

**Reply by karolkielecki** [reply]
Hi @wellzenon, 
It's not supported. You'll need to create another plugin based on OIDC.

**Reply by wellzenon** [reply]
Oh, thank you
